### PR TITLE
Add kwarg to sort CVSS JSON dict before returning it

### PR DESCRIPTION
**Pull request by juspence**
In some cases you can use json.dumps(my_dict, sort_keys=True), but this PR will support cases where the Python dict should be sorted while remaining a Python dict (so is not dumped to JSON format).

### CODE REVIEW

**Comment by skontar** [comment]
Hi! Thanks for the contribution.

Please write tests to cover the new functionality. 

Also I believe that dictionaries maintain order only starting with Python 3.7, so I think it should check for Python version and if it is <3.7 throw `NotImplementedError` or maybe return `OrderedDict`?

File: cvss/constants2.py [hunk 1]
```diff
@@ -107,3 +109,6 @@
                                                        ('H', 'High'),
                                                        ('ND', 'Not Defined')])),
                                    ])
+
+# '3.6.8', '3.9.6', etc.
+PYTHON_VERSION = sys.version.split()[0]
```
Comment (skontar) [inline cvss/constants2.py:114]: Seems like the following is a better approach to get the same string?

```
>>> import platform
>>> platform.python_version()
'3.9.9'
```

**Review by skontar — CHANGES_REQUESTED** [review]

File: cvss/cvss2.py [hunk 2]
```diff
@@ -340,5 +344,10 @@ def us(text):
             data['temporalScore'] = float(self.temporal_score)
         if self.environmental_score:
             data['environmentalScore'] = float(self.environmental_score)
+        if sort:
+            if PYTHON_VERSION < '3.7':
```
[the remark this answers is not in the document]
Comment (skontar) [reply cvss/cvss2.py:348]: Either use `LooseVersion`:
```
>>> from distutils.version import LooseVersion
>>> LooseVersion("3.60.0") < LooseVersion("3.7")
False
```
Or use `tuple` with `int` comparison:
```
>>> import sys
>>> pvi = sys.version_info.major, sys.version_info.minor
>>> pvi
(3, 9)
>>> pvi < (3, 7)
False
>>> pvi = (3, 60)
>>> pvi < (3, 7)
False
>>> pvi = (3, 6)
>>> pvi < (3, 7)
True
```

[the remark this answers is not in the document]
Comment (juspence) [reply cvss/cvss2.py:348]: Hmm, what do you think about always returning an OrderedDict when the user wants to sort the output, no matter what the version is? That way we don't need the same complex logic in two different places (cvss2.py and cvss3.py)

[the remark this answers is not in the document]
Comment (skontar) [reply cvss/cvss2.py:348]: If the only purpose is to generate an object to then serialize using `json.dumps`, then surely it will work as intended.

**Comment by skontar** [comment]
@mprpic, are you planning to review this change?

**Review by mprpic — APPROVED** [review]
LGTM!

**Pull request merged**